ApplicationLibCode/Application/RiaResultNames.cpp

```diff
@@ -313,6 +313,30 @@ QString RiaResultNames::formationAllanResultName()
     return "Formation Allan";
 }
 
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+QString RiaResultNames::indexIResultName()
+{
+    return "INDEX_I";
```

Comment:
@magnesj Prefixed with INDEX to get the next to eachother in the property editor. Feel free to suggest other names!